vunit/sim_if/ghdl.py

```diff
@@ -382,10 +391,19 @@ def simulate(self, output_path, test_suite_name, config, elaborate_only):  # pyl
         except Process.NonZeroExitCode:
             status = False
 
+        if config.sim_options.get(self.name + ".gtkwave_script.gui", None):
+            warn_str = (
+                "%s.gtkwave_script.gui is deprecated and will be removed " % self.name  # pylint: disable=C0209
```

Comment:
Is there a reason for why f-strings can't be used so that we don't have to suppress C0209?

Reply:
There were linter warnings about that (but I didn't do it in a proper way either), that I couldn't get rid of (either that or too long lines).

The reason is that one should call loggers with `warning("%s", foo)` so that the string is formatted upon calling warning. 

Reminding me about how it should be done, the correct approach is now used.